### PR TITLE
Celery beat and scheduling

### PR DESCRIPTION
**Pull request by oscar-sportamore**

### CODE REVIEW

File: blues/application/tasks.py [hunk 1]
```diff
@@ -259,6 +260,18 @@ def configure_environment():
 
 
 @task
+def configure_beat_schedule():
+    from .project import project_name, git_repository_path
+
+    beat_schedule = blueprint.get('beat_schedule', None)
+    if beat_schedule:
+        blueprint.upload('beat/schedule',
+                         os.path.join(git_repository_path(), '.beat_schedule'),
```
Comment (carlskeide) [inline blues/application/tasks.py]: Placing non-standard files in the git path is risky, several tasks will perform hard resets, so every project that implements beat would have to add it to their gitignore (unlike `.env`, which is usually in the default ignore file)

File: blues/application/tasks.py [hunk 2]
```diff
@@ -259,6 +260,18 @@ def configure_environment():
 
 
 @task
+def configure_beat_schedule():
+    from .project import project_name, git_repository_path
+
+    beat_schedule = blueprint.get('beat_schedule', None)
```
Comment (carlskeide) [inline blues/application/tasks.py]: I feel it would be more consistent with other app.settings to name this "beat" or even "beat: schedule"

**Comment by carlskeide** [comment]
I was going to ask about the "with" thing, but you'd already removed it

**Comment by oscar-sportamore** [comment]
Yeah, seems `with` has to be explicitly imported in this version of jinja.

**Pull request merged**